source/_integrations/smtp.markdown

```diff
@@ -85,7 +85,7 @@ A sample configuration entry for Google Mail.
 notify:
   - name: "NOTIFIER_NAME"
     platform: smtp
-    server: "smtp.example.com"
+    server: "smtp.gmail.com"
```

Comment:
Please use `example.com` is it designed/documented to use for examples by RFC2606

Reply:
I will do that and reword the text that comes before the example to make it clear that this is not a google mail example but a generic SMTP example.